pkg/accounting/tally/tally.go

```diff
@@ -71,7 +71,7 @@ func (t *Service) Tally(ctx context.Context) error {
 		errAtRest = errs.New("Query for data-at-rest failed : %v", err)
 	} else {
 		if len(nodeData) > 0 {
-			err = t.SaveAtRestRaw(ctx, latestTally, time.Now().UTC(), nodeData)
+			err = t.SaveStoragenodeStorageTallies(ctx, latestTally, time.Now().UTC(), nodeData)
```

Comment:
Any chance we can try to format `SaveStoragenodeStorageTallies` and `SaveBucketTallies`  the same?  

```
if len(nodeData) > 0 {
    err = t.SaveStoragenodeStorageTallies(ctx, latestTally, time.Now().UTC(), nodeData)
    if err != nil {
        errAtRest = errs.New("Saving storage node data-at-rest failed : %v", err)
    }
}
if len(bucketData) > 0 {
    _, err = t.accountingDB.SaveBucketTallies(ctx, latestTally, bucketData)
    if err != nil {
        errBucketInfo = errs.New("Saving bucket storage data failed")
    }
}
```